lang/py/test/test_schema.py

```diff
@@ -575,69 +582,19 @@ def test_exception_is_not_swallowed_on_parse_error(self):
 
     self.assertTrue(caught_exception, 'Exception was not caught')
 
-  def test_decimal_invalid_schema(self):
```

Comment:
Why is this test case removed? What and how it tests is still valid: schema parsing should fail for negative precision, scale etc.

Reply:
ah okay, never mind, started the review of this file from the wrong side. :) This is now refactored to `test_parse_invalid`, if I'm  not mistaken.